identifier-ui/.gitignore

```diff
@@ -12,7 +12,8 @@
 /build
 
 # misc
-.DS_Store
+package-lock.json
```

Comment:
we should keep the lockfile checked into source control so we're getting same versions of packages: [stackoverflow](https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5)

Reply:
```suggestion
```